scripts/reactor_deployment.py

```diff
@@ -359,7 +425,7 @@ def rand_deployment(df, base_col, ar_dict, dep_start_year,
     # initialize the number of reactor columns
```

Comment:
Is this comment helping you understand or simply stating what the function is doing? If it's the latter, I should be able to look at the docstring of the function to get that information. Food for thought.
```suggestion
```